src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -33,10 +35,20 @@ private static void RunDataTestForSingleConnString(string tcpConnectionString)
             BasicConnectionPoolingTest(tcpConnectionString);
             ClearAllPoolsTest(tcpConnectionString);
             ReclaimEmancipatedOnOpenTest(tcpConnectionString);
+            SoftTcpShutdownConnectionTest(tcpConnectionString);
```

Comment:
Can we have these tests run on their own instead of running them all together under 1 test case like that?
The TvpTest design style is pretty old and gets cumbersome to figure out issues if 1 of those test fail.. It also blocks other tests from running if the first one fails.